bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableConnection.java

```diff
@@ -105,6 +105,20 @@ public AbstractBigtableConnection(Configuration conf) throws IOException {
     this(conf, false, null, null);
   }
 
+  /**
+   * Default constructor called from {@link ConnectionFactory#createConnection(Configuration)} and
+   * in its many forms via reflection.
+   * 
+   * @param conf The configuration of 
+   * @param managed This should always be false. It's an artifact of old HBase behavior.
```

Comment:
That is what I thought. LGTM.
On Jan 14, 2016 7:14 AM, "Solomon Duskis" <notifications@github.com> wrote:

> In
> bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableConnection.java
> <https://github.com/GoogleCloudPlatform/cloud-bigtable-client/pull/628#discussion_r49718871>
> :
>
> > @@ -105,6 +105,20 @@ public AbstractBigtableConnection(Configuration conf) throws IOException {
> >      this(conf, false, null, null);
> >    }
> >
> > +  /**
> > +   * Default constructor called from {@link ConnectionFactory#createConnection(Configuration)} and
> > +   * in its many forms via reflection.
> > +   *
> > +   * @param conf The configuration of
> > +   * @param managed This should always be false. It's an artifact of old HBase behavior.
>
> Unfortunately, not. ConnectionFactory.createConnection(Configuration) uses
> reflection and looks for this signature.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/cloud-bigtable-client/pull/628/files#r49718871>
> .
>